routing/routes_builder/routes_builder_tool/routes_builder_tool.cpp

```diff
@@ -0,0 +1,108 @@
+#include "routing/routes_builder/routes_builder_tool/utils.hpp"
+
+#include "routing/routes_builder/routes_builder.hpp"
+
+#include "base/assert.hpp"
+#include "base/logging.hpp"
+
+#include <exception>
+#include <string>
+#include <vector>
+
+#include "3party/gflags/src/gflags/gflags.h"
+
+DEFINE_uint64(threads, 0,
+              "The number of threads. std::thread::hardware_concurrency() is used by default.");
+
+DEFINE_string(routes_file, "", "Path to file with routes in format: \n\t"
+                               "first_start_lat first_start_lon first_finish_lat first_finish_lon\n\t"
+                               "second_start_lat second_start_lon second_finish_lat second_finish_lon\n\t"
+                               "...");
+
+DEFINE_string(dump_path, "", "Path where routes will be dumped after building."
+                             "Useful for intermediate results, because routes building "
+                             "is a long process.");
+
+DEFINE_string(data_path, "", "Path to dir with mwms.");
+
+DEFINE_string(api_name, "", "Api name, current options: mapbox,google");
+DEFINE_string(api_token, "", "Token for chosen api.");
+
+DEFINE_int64(start_from, 0, "The line number from which the tool should start read.");
```

Comment:
`DEFINE_int64(start_from, 0, "The line number from which the tool should start reading.");`